crates/holo_hash/src/tests.rs

```diff
@@ -38,7 +38,7 @@ async fn check_hashed_type() {
 }
 
 #[test]
-#[ignore]
+#[ignore = "TODO"]
```

Comment:
o_O - I was expecting it to print out the string literal along with the `ignored` on `cargo test`... but apparently it just ignores the string?

Reply:
Yeah exactly, I don't quite see the point of this syntax...but at least this makes it easier to grep for ignored tests and see why they're ignored on the same line.